packages/loaders/src/styled/StyledProgress.ts

```diff
@@ -30,15 +30,34 @@ interface IStyledProgressBackgroundProps {
 
 const PROGRESS_BACKGROUND_COMPONENT_ID = 'loaders.progress_background';
 
+const colorStyles = ({
+  theme,
+  color
+}: IStyledProgressBackgroundProps & ThemeProps<DefaultTheme>) => {
+  const backgroundColor = getColor({
+    theme,
+    hue: 'neutralHue',
+    transparency: theme.opacity[200],
+    light: { shade: 700 },
+    dark: { shade: 500 }
+  });
+  const foregroundColor = color || getColor({ theme, variable: 'border.successEmphasis' });
```

Comment:
> OK, so we can only review color values, and we are counting on eng to implement correct variables (e.g. progress loaders might use bg group, not border and it should be 100% clear from the inventory)?

@lucijanblagonic I always define colors using the variables found in the inventory first.

For `Progress bar`, the `foregroundColor` uses `border.successEmphasis`. It exists in our collection of border variables so I used it.

The `backgroundColor` uses `WIP_opacity/neutral (700 : 500)/200`, which is the same as `foreground.subtle`. However, because we don't have a background variable that matches the hue / shade combo per mode, I build it using advanced `getColor` options.

I hope this answers your question. If this isn't the right approach, let me know. 

Thanks!